CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased
+- Fix message for box command on default help @budhrg
```

Comment:
We are pushing the new changelogs at the bottom of Unreleased section to avoid the conflicts, no?